modules/machine-api-overview.adoc

```diff
@@ -32,21 +26,23 @@ specified `MachineSet`, and the `MachineAutoscaler` maintains that range of node
 The `MachineAutoscaler` object takes effect after a `ClusterAutoscaler` object
 exists. Both `ClusterAutoscaler` and `MachineAutoscaler` resources are made
 available by the `ClusterAutoscalerOperator`.
-`ClusterAutoscaler`:: This resource is based on the upstream
-link:https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler[ClusterAutoscaler]
+
+`ClusterAutoscaler`:: This resource is based on the upstream ClusterAutoscaler
 project. In the {product-title} implementation, it is integrated with the
 Cluster API by extending the `MachineSet` API. You can set cluster-wide
 scaling limits for resources such as cores, nodes, memory, GPU,
 and so on. You can set the priority so that the cluster prioritizes pods so that
 new nodes are not brought online for less important pods. You can also set the
 ScalingPolicy so you can scale up nodes but not scale them down.
+
 `MachineHealthCheck`:: This resource detects when a machine is unhealthy,
 deletes it, and, on supported platforms, makes a new machine.
 
 
-In {product-title} version 3.11, you could not roll out a multi-zone architecture easily because the cluster
-did not manage machine provisioning. It is easier in 4.0. Each `MachineSet` is scoped
-to a single zone, so the installation program sends out `MachineSets` across availability zones
-on your behalf. And then because your compute is dynamic, and in
-the face of a zone failure, you always have a zone for when you need to rebalance
-your machines. The autoscaler provides best-effort balancing over the life of a cluster.
+In {product-title} version 3.11, you could not roll out a multi-zone
+architecture easily because the cluster did not manage machine provisioning. It
+is easier in 4.0. Each `MachineSet` is scoped to a single zone, so the
```

Comment:
This likely shouldn't be `4.0`. It might make sense to have `product-version` here, but I wonder if we could also reword this sentence to future-proof it. What about the following:

  Beginning with 4.1 this process is easier. Each `MachineSet` is scoped to a single zone...